src/components/Login/roleForm.css

```diff
@@ -0,0 +1,31 @@
+.text-align-left {
+    text-align: left;
+}
+
+.section-header-spacing {
+    color: #2b388f;
```

Comment:
why this is not part of the theme?

Reply:
It's specific to one form but it's a good idea. I'll post-pone when the ticket for the theme will be done.